modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialOpenApiGenerator.java

```diff
@@ -375,10 +375,10 @@ private String getApiDocsJson(CredentialVariables credentialVariables) {
                                                 "example": "%s",
                                                 "description": "The name of the credential"
                                             },
-                                            "userName": {
+                                            "userId": {
```

Comment:
do we need the userId here? Its better if userId logic is completely transparent to the user.

Reply:
OK, I will remove it.